lib/fun_with_flags/ui/router.ex

```diff
@@ -290,4 +292,9 @@ defmodule FunWithFlags.UI.Router do
     csrf_token = Plug.CSRFProtection.get_csrf_token()
     Plug.Conn.assign(conn, :csrf_token, csrf_token)
   end
+
+
+  defp protect_from_forgery(conn, opts \\ []) do
+    Plug.CSRFProtection.call(conn, Plug.CSRFProtection.init(opts))
```

Comment:
I know that this is just copying what [the Phoenix function does](https://github.com/phoenixframework/phoenix/blob/v1.3.4/lib/phoenix/controller.ex#L951), but since [`Plug.CSRFProtection.init/1`](https://github.com/elixir-plug/plug/blob/v1.6.3/lib/plug/csrf_protection.ex#L212-L217) doesn't need to access any dynamic data, maybe this could be replaced with:

```elixir
plug Plug.CSRFProtection
```

so that `init` is called once instead of with each request.

I assume that in Phoenix this is implemented like that in order to expose is as the `protect_from_forgery` helper, but now I wonder if there is another reason.

Reply:
With what [I suggested](https://github.com/tompave/fun_with_flags_ui/pull/6#issuecomment-427760993) in mind, it would probably be easier to do it if you disregard my suggestion above.